src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -27,6 +29,40 @@ public void equals_False_X10() {
     assertNotEquals(rnX, i);
   }
 
+  @Test
+  public void equals_True_Reflexive() {
+    assertTrue(rnX.equals(rnX));
+  }
+
+  @Test
+  public void equals_True_Transitive() {
+    RomanNumeral x = new RomanNumeral("IV");
+    RomanNumeral y = new RomanNumeral("IV");
+    RomanNumeral z = new RomanNumeral("IV");
+
+    assertTrue(x.equals(y) && y.equals(z) && x.equals(z));
+  }
+
+  @Test
+  public void equals_True_Symmetric() {
+    RomanNumeral x = new RomanNumeral("IV");
+    RomanNumeral y = new RomanNumeral("IV");
+
+    assertTrue(x.equals(y) && y.equals(x));
+  }
+
+  @Test
+  public void equals_True_StableOverTime() {
+    RomanNumeral x = new RomanNumeral("X");
+    assertTrue(x.equals(rnX));
+    assertTrue(x.equals(rnX));
+  }
+
+  @Test
+  public void equals_False_Null() {
+    RomanNumeral x = new RomanNumeral("IV");
+    assertFalse(x.equals(null));
+
```

Comment:
I think there's a missing closing curly brace. Remember to compile your code before committing..

Reply:
Sorry.

Reply:
I thought I had that closed. But I must have missed it prior to my commit.

Reply:
It happens. Don't feel bad. Reviewers love catching things like that.